docs/UserGuide.adoc

```diff
@@ -150,7 +150,7 @@ A member can have any number of tags (including 0)
 Examples:
 
 * `memberAdd n/John Doe mn/A0123456Z p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 g/Male
-y/Year 2 m/Chemistry t/Swimming`
+ y/Year 2 m/Chemistry t/Swimming`
```

Comment:
Is double space necessary?

Reply:
not exactly a double space, the newline is showed as a space, but the output only has 1 space